.eslintrc.json

```diff
@@ -139,23 +139,31 @@
         "node/no-unsupported-features/es-syntax": "off"
       }
     },
+    {
+      "files": ["**/bin/**/*.js"],
+      "rules": { "node/shebang": "off" }
+    },
     {
       "files": [
         "**/bin/**/*.js",
-        "**/spec/**/*.js",
+        "example/spec/**/*.js",
         "packages/api/migrations/**/*.js",
         "packages/deployment/**/*.js",
         "tf-modules/internal/cumulus-test-cleanup/**/*.js"
       ],
-      "rules": { "no-console": "off" }
+      "rules": {
+        "no-console": "off",
+        "node/no-extraneous-require": "off"
+      }
     },
     {
       "files": ["**/test/**/*.js", "**/tests/**/*.js"],
       "rules": {
         "max-classes-per-file": "off",
         "no-console": "off",
         "no-new": "off",
-        "no-param-reassign": "off"
+        "no-param-reassign": "off",
+        "node/no-extraneous-require": "off"
```

Comment:
I'm presuming this is due to the vscode linter getting angry in our test files due to pulling the dependencies up to the root level?    That seems fine...but....

In thinking about it, this is going to essentially make our individual packages unbuildable outside a full bootstrap.  I don't know that I mind given the build time impacts, but as much as I hate the boilerplate we should update the READMEs.

Reply:
It wasn't VS Code complaining, it was eslint. The issue was caused because we have packages that the test files rely on that are not defined in their immediate `package.json`. Ideally, the `node/no-extraneous-require` rule could be told, "hey, check this other package.json file as well", but it doesn't have that flexibility.

As for making individual packages unbuildable outside of a full bootstrap, maybe, but I don't have a problem with that. We have already been performing some operations at a repo level anyway. `eslint`, for example, is configured at the repo level. We don't support running eslint in an individual package.

As for updating the READMEs, this does seem like something that should be noted in the top-level developer docs. Something like, "this is a monorepo, not a collection of independent packages co-located in a git repo. It builds like a monorepo." But nicer.

Reply:
Disabling `node/no-extraneous-require` seems like a pretty bad change. Then when you write `require` statements into your code referencing modules that you never installed, you don't notice. And then the CI linter doesn't even catch it and you only get random CI test failures as a clue.

Regarding building individual packages, running `npm run package` in `packages/api` still seemed to work for me? If that actually is broken somehow, I'm less keen on these changes. Re-building just the API package to deploy API lambda changes is a common occurrence for me.